src/api/python/z3compat/z3.py

```diff
@@ -0,0 +1,255 @@
+############################################
+# Copyright (c) 2021 The CVC5 Developers
+#               2012 The Microsoft Corporation
+#
+# CVC5's Z3-based Python interface
```

Comment:
@mpreiner caught elsewhere that it's `cvc5` not `CVC5` so I'm flagging it here also.

Reply:
Done in f774ebf5ac